docs/Modding/Tilesets/nations.rst

```diff
@@ -0,0 +1,41 @@
+Adding a Nation
```

Comment:
This is already contained in https://longturn--795.org.readthedocs.build/en/795/Modding/Rulesets/nations.html, although I fully support having something in the tilesets section too. Maybe the modding docs should be reorganized as "How to" (with tutorials and quick guides) and "Reference" (in-depth, dry reference) but certainly not now.

Reply:
I totally forgot that I wrote that one. I will remove one of them and then we can figure out a better way of organizing the modding section.